code/modules/supermatter/supermatter.dm

```diff
@@ -368,6 +368,15 @@
 
 
 /obj/machinery/power/supermatter/attack_hand(mob/user as mob)
+	var/obj/item/clothing/gloves/golden/G = user.get_item_by_slot(slot_gloves)
+	if(istype(G))
+		to_chat(user,"<span class='warning'>Carefully extending a single finger, you gently touch the supermatter.</span>")
```

Comment:
I would change the "Carefully extending a single finger, you gently touch the supermatter." line to "Carefully extending a single finger you close in to touch the supermatter, but are stopped by a tingling feeling coming from the gloves." or something along those lines for consistency with supermatter behaviour.